### PR TITLE
[new] zingtouch@1.0.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by creativetim**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/zingchart/zingtouch
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/zingtouch
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 735
   - Count of watchers: 26
   - Count of forks: 33
 * NPM download stats (optional):
   - Downloads in the last day: 5
   - Downloads in the last week: 55
   - Downloads in the last month: 55

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

--

I downloaded the file via the repo's git repo and copied to my local cdnjs feature branch.

*NOTE: This is my first time adding something to cdnjs. I apologize if I've not checked something I should have in the PR template.*

### CODE REVIEW

File: ajax/libs/zingtouch/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "keywords": [
+    "zingtouch",
+    "touch",
+    "javascript"
+  ],
+  "author": {
+    "name": "ZingChart"
+  },
+  "homepage": "https://github.com/zingchart/zingtouch",
```
Comment (PeterDaveHello) [inline ajax/libs/zingtouch/package.json]: https://zingchart.github.io/zingtouch/ will be better.

**Comment by PeterDaveHello** [comment]
Please also add the license info in package.json, thanks.

File: ajax/libs/zingtouch/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "git+https://github.com/zingchart/zingtouch.git"
+  },
+  "keywords": [
+    "zingtouch",
+    "touch",
+    "javascript"
+  ],
+  "author": {
+    "name": "ZingChart"
+  },
```
Comment (pvnr0082t) [inline ajax/libs/zingtouch/package.json]: Let's use `"author": "ZingChart",` here.

**Comment by creativetim** [comment]
:+1: Updated homepage, author, and added license.

@PeterDaveHello @pvnr0082t

**Comment by mike-schultz** [comment]
@creativetim ZingTouch author here. Thanks for adding this in here!

**Comment by creativetim** [comment]
Awesome! It's my pleasure @mike-schultz; thank you for building such a great lib!

**Comment by PeterDaveHello** [comment]
I changed the commit message a little bit to be much more clean, thanks for your contribution.

**Pull request merged**